src/components/commoncomponents/header/index.js

```diff
@@ -1,25 +1,21 @@
 import React from 'react';
-import { Text, StyleSheet, TouchableOpacity, Image } from 'react-native';
-import { widthPercentageToDP as wp, heightPercentageToDP as hp} from 'react-native-responsive-screen';
-
-
+import { TouchableOpacity, Image } from 'react-native';
+import styles from './styles'
 
+/*
+  Header component for go back functionality
+*/
 function Header(props) {
     const { onClick } = props;
     return (
         <TouchableOpacity
-        style={{backgroundColor: 'transparent', marginTop: wp("3"), marginBottom: wp("3")}}
+        style={styles.container}
         onPress={() => onClick()}>
```

Comment:
Never use inline functions. Create a function and pass that function call here in onPress.